templates/zerver/for/open-source.md

```diff
@@ -1,104 +1,92 @@
+hackathons, meetups, and others. If you’re not sure whether your
+organization qualifies, send us an email at support@zulipchat.com.
+
+**Scale to thousands of users**
+
+For those running Zulip at home, we’ve done a lot of work to ensure that
+drive-by members of the community don’t consume too many disk or CPU
+resources. A Zulip server with 1000 active members and 10000 inactive
+members takes about as many resources as a Zulip server with 1000 active
+members.
+
+**Join without an invitation**
+
+Allow anyone to
+[join without an invitation](/help/allow-anyone-to-join-without-an-invitation).
+You can also link to your Zulip with a [badge](/help/join-zulip-chat-badge)
+in your readme document.
+
+**Authenticate with GitHub**
+
+Allow (or require) users to
+[authenticate with their GitHub account][github-auth], instead of with their
+email address.
 [github-auth]: https://github.com/zulip/zulip/blob/7e9926233/zproject/prod_settings_template.py#L112
 
-* **Scale:** Zulip scales well to large communities — it's loved by
-  communities as large as 1000 users and beyond.
+**Quality data export**
+
+Our high quality export and import tools ensure you can always move from
+[zulipchat.com](https://zulipchat.com) hosting to your own servers.
+
+**Syntax highlighting**
+
+[Full Markdown support](/help/format-your-message-using-markdown), including
+syntax highlighting, makes it easy to discuss code, paste an error message,
+or make a complicated argument. Full LaTeX support as well.
+
+**Permalink from issues to chat**
+
+[Permalink to Zulip conversations](/help/share-a-message-or-conversation)
+from your issue tracker. Zulip’s topic-based threading keeps conversations
+coherent and organized.
+
+**Link from chat to issues**
+
+Efficiently refer to issues or code reviews with notation like `#1234` or
+`T1234`. You can set up any regex as a
+[custom linkification filter](/help/add-a-custom-linkification-filter) for
+your organization.
 
-## Zulip ❤​ open source
+**Hundreds of integrations**
 
-* **Free and open source software:** Zulip itself is open source,
-  [under the Apache License 2.0](https://github.com/zulip/zulip/blob/master/LICENSE).
+Get events from GitHub, Travis CI, JIRA, and
+[hundreds of other tools](/integrations) right in Zulip. Topics give each
+issue its own place for discussion.
 
-* **Free for open source:** Full, commercially-supported hosting at
-  zulipchat.com is free for open source communities.  [Sign up
-  today!](https://zulipchat.com/beta/))
+**Free and open source**
 
-<!--- This needs better — any! — documentation. -->
-* **Your data is yours:** Zulip's high-quality export and import tools
-  enable migrating a community smoothly from one host to another.
-  So if you start out running on zulipchat.com, you can always choose
-  later to move to your own servers, and vice versa.
+Don’t like something? You can open an issue, submit a patch, fork the
+project, or chat with us directly at
```

Comment:
Maybe link somewhere to the repo and/or contribution instructions. Those aren't super easy to find from zulipchat.com right now in general (which we should fix, but also we can help when we're talking directly about it already.)

Maybe even link each action to the issues page, a how-to-contribute doc, and the repo respectively.